cuFOLIO_portfolio_optimization/CVaR/01_optimization_with_cufolio.ipynb

```diff
@@ -40,10 +40,10 @@
     "# If dependencies are already installed, you can comment out or skip this cell.\n",
     "\n",
     "# Install cuOpt (if not already installed)\n",
-    "#!pip install --upgrade --user --extra-index-url https://pypi.nvidia.com -q cuopt-cu12\n",
+    "# !pip install --upgrade --user --extra-index-url https://pypi.nvidia.com -q cuopt-cu12==25.10.* \n",
```

Comment:
haha I removed the spaces on purpose so that a user would only have to delete the # and not two characters

Reply:
do we want to be explicit, or do we want it always to grab the latest?

Reply:
Ahh, I think I might have copied from somewhere and it caused issue.